Mute fail (#16)
smeup/smeup-rpg: docs/mute.md

SUCCESS
```

> There is also an utility to run just one file with the MUTE engine. To create the appropriate jar, run

Reviewer: ```suggestion
It is also possible to package the MUTE engine as a single jar that can be run independently. To create such jar, yoi can run:
```

```
 ./gradlew fatMuteJar
 ```

> This command creates `rpgJavaInterpreter-core/build/libs/rpgJavaInterpreter-core-mute-all.jar`

Reviewer: ```suggestion
This command creates the JAR `rpgJavaInterpreter-core/build/libs/rpgJavaInterpreter-core-mute-all.jar`.
```

> Then run the mute test with:

Reviewer: ```suggestion
At this point you can run MUTE tests with:
```
